wom/src/main/scala/wdl4s/wdl/Declaration.scala

```diff
@@ -80,6 +84,8 @@ trait DeclarationInterface extends WdlGraphNodeWithUpstreamReferences {
 
 object Declaration {
 
+  type DeclarationNode = Either[ExpressionNode, GraphInputNode]
```

Comment:
After this morning's discussion, I'm going to leave this as-is for now, but I'll bear this in mind if/when we need to add another type.